Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apt install ffmpeg -y
 
 # Install requirements first to take advantage of docker build layer caching
 COPY ./src/requirements.txt /tmp/requirements.txt
+RUN pip install --upgrade pip
```

Comment:
Do you have a specific issue that this solves? There's not really any reason to do this inside the docker image, it will just increase build time and inflate the final size.

Reply:
Latest version of pip isn't really needed unless some of the dependencies have install issues without it (it's never invoked again after this)